app/views/landing-pages/league/PageLeagueGlobal.vue

```diff
@@ -55,11 +55,15 @@ export default {
     previousChampionshipArenaSlug: previousChampionshipArena ? previousChampionshipArena.slug : null,
     regularArenaSlug: currentRegularArena ? currentRegularArena.slug : 'mages-might',
     championshipArenaSlug: currentChampionshipArena ? currentChampionshipArena.slug : null,
-    championshipActive: !!currentChampionshipArena
+    championshipActive: !!currentChampionshipArena,
+    anonymousPlayerName: false
   }),
 
   beforeRouteUpdate (to, from, next) {
     this.clanIdOrSlug = to.params.idOrSlug || null
+    if(this.clanIdOrSlug) {
+      this.anonymousPlayerName = features.enableAnonymization
```

Comment:
where is this features defined?

Reply:
oh actually defined in server code 😂  client feature init would be overwrite by server feature. 😂  our stupid feature cross client & server